src/praisonai-agents/praisonaiagents/goal/engineer.py

```
@@ -0,0 +1,192 @@
+        score = 0.0
+        reasoning = ""
+        try:
+            from ..eval import Judge
+
+            judge_result = Judge(
+                model=self.config.model,
+                criteria=criteria_text,
+            ).run(output=output)
+            score = float(getattr(judge_result, "score", 0.0) or 0.0)
+            reasoning = getattr(judge_result, "reasoning", "") or ""
+        except Exception as exc:  # pragma: no cover - network/optional dep
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)